xrpl_types/src/types.rs

```diff
@@ -1,9 +1,9 @@
-mod amount;
-mod currency;
-mod currency_code;
-mod ledger_timestamp;
-mod primitive;
-mod transaction;
+pub mod amount;
```

Comment:
This means that types have two "paths". Does not seem like a good idea

Reply:
I think the primitives should be exported since they can be used as building blocks.
In fact that was the reason i made them public, since this is a library - it's very probably that the lower level primitives will be composed to create new functionality.
In this case, making them private and inaccessible makes the library difficult to deal with.

Reply:
Yep good point, this was not needed. Thanks 👍